hknweb/candidate/admin.py

```diff
@@ -1,26 +1,44 @@
 from django.contrib import admin
-from .models import OffChallenge, Announcement
 from django.contrib.auth.models import User
 from django.http import HttpResponse
 
+from .models import OffChallenge, Announcement
+from .views import send_cand_confirm_email
+
 import csv
 
 
 class OffChallengeAdmin(admin.ModelAdmin):
 
-    fields = ['requester', 'officer', 'name', 'reviewed', 'confirmed', 'description', 'proof', 'officer_comment', 'request_date']
+    fields = ['requester', 'officer', 'name', 'officer_confirmed', 'csec_confirmed', 'description', 'proof', 'officer_comment', 'request_date']
     readonly_fields = ['request_date']
-    list_display = ('name', 'requester', 'officer', 'reviewed', 'confirmed', 'request_date')
-    list_filter = ['requester', 'officer', 'request_date']
+    list_display = ('name', 'requester', 'officer', 'officer_confirmed', 'csec_confirmed', 'request_date')
+    list_filter = ['requester', 'officer', 'officer_confirmed', 'csec_confirmed', 'request_date']
     search_fields = ['requester', 'officer', 'name']
 
-    actions = ["export_as_csv"]
+    actions = ["export_as_csv", "csec_confirm", "csec_reject"]
 
     def formfield_for_foreignkey(self, db_field, request, **kwargs):
         if db_field.name == "officer":
             kwargs["queryset"] = User.objects.all().order_by('username')
         return super(OffChallengeAdmin, self).formfield_for_foreignkey(db_field, request, **kwargs)
 
+    def save_model(self, request, obj, form, change):
+        super().save_model(request, obj, form, change)
+        if 'csec_confirmed' in form.changed_data:
+            OffChallengeAdmin.check_send_email(request, obj)
+
+    @staticmethod
+    def check_send_email(request, obj):
+        # officer has already confirmed, and now csec confirms
+        if obj.csec_confirmed is True and obj.officer_confirmed is True:
```

Comment:
`is True` is rather unnecessary, and readability is preserved without it:
```py
if obj.csec_confirmed and obj.officer_confirmed
```
In particular, None is falsey.

Reply:
I find this more intuitive since this is more like a three state enum rather than a boolean